<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QMainWindow" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainFrame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>651</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupCodec">
       <property name="title">
        <string>Кодировка</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioCP">
           <property name="text">
            <string>cp1251</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioUTF">
           <property name="text">
            <string>UTF-8</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupStyle">
       <property name="title">
        <string>Начертание</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>171</width>
          <height>65</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="boxBold">
           <property name="text">
            <string>Полужирный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="boxItalic">
           <property name="text">
            <string>Курсив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="boxUnder">
           <property name="text">
            <string>Подчеркнутый</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupOther">
       <property name="title">
        <string>Другое</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Черный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Красный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зеленый</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Синий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Серый</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Настройки по умолчанию</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить как...</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>214</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>9</y>
      <width>215</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>9</y>
      <width>214</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="Mainmenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menumain"/>
  </widget>
  <action name="action_help">
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>radioCP</tabstop>
  <tabstop>radioUTF</tabstop>
  <tabstop>boxBold</tabstop>
  <tabstop>boxUnder</tabstop>
  <tabstop>boxItalic</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButtonSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
